ilc/server/tailor/fetch-template.spec.js

```diff
@@ -0,0 +1,64 @@
+const chai = require('chai');
+const sinon = require('sinon');
+
+const fetchTemplates = require('./fetch-template');
+
+describe('fetch templates', () => {
+  const configsInjector = {
+    inject: sinon.stub(),
+  };
+
+  const newrelic = {
+    setTransactionName: sinon.spy(),
+  };
+
+  const registryService = {
+    getTemplate: (arg) => arg,
```

Comment:
I think this function should be async to properly mock the behaviour of the original one